readthedocs/docsitalia/models.py

```diff
@@ -241,6 +253,13 @@ def active_documents(self):
             '-modified_date', '-pub_date'
         )
 
+    def description(self):
+        """Get publisher project description from metadata"""
+        try:
+            return self.metadata['description']
```

Comment:
why not `return self.metadata.get('description', '')` instead of catching the exception

Reply:
looks better indeed, fixed